hadoop/src/test/java/org/apache/carbondata/hadoop/test/util/StoreCreator.java

```diff
@@ -465,6 +466,7 @@ private static void generateGraph(IDataProcessStatus schmaModel, SchemaInfo info
     model.setEscapeCharacter(schmaModel.getEscapeCharacter());
     model.setQuoteCharacter(schmaModel.getQuoteCharacter());
     model.setCommentCharacter(schmaModel.getCommentCharacter());
+    model.setTrim(schmaModel.getTrim());
```

Comment:
Not clear about the usecase, can you make it more clear by providing more details